### PR TITLE
Remove RawGit

### PR DESCRIPTION
**Pull request by bcdunbar**
As part of https://github.com/plotly/streambed/issues/11637

remove rawgit and 🚚 assets.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 if 'DYNO' in os.environ:
     app.scripts.append_script({
-        'external_url': 'https://cdn.rawgit.com/chriddyp/ca0d8f02a1659981a0ea7f013a378bbd/raw/e79f3f789517deec58f41251f7dbb6bee72c44ab/plotly_ga.js'
+        'external_url': 'https://codepen.io/plotly/pen/BGyZNa.js'
```
Comment (xhlulu) [inline app.py]: Is this for a temporary fix? I don't think linking to codepen is good long term since it's not persistent.

File: app.py [hunk 2]
```diff
@@ -37,9 +37,7 @@
     "https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.min.css",
     "https://cdnjs.cloudflare.com/ajax/libs/skeleton/2.0.4/skeleton.min.css",
     "//fonts.googleapis.com/css?family=Raleway:400,300,600",
-    "https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css",
-    "https://cdn.rawgit.com/plotly/dash-tsne/master/custom_styles.css",
-    "https://cdn.rawgit.com/plotly/dash-app-stylesheets/2cc54b8c03f4126569a3440aae611bbef1d7a5dd/stylesheet.css"
+    "https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
```
Comment (xhlulu) [inline app.py]: We could add this to the asset too? Now or in a future PR it's fine.

**Comment by xhlulu** [comment]
@bcdunbar can you give me read permission to plotly/streambed#11637?

File: app.py (hunk 1 above)
Comment (bcdunbar) [reply app.py]: 👍  added to `/assets`

File: app.py (hunk 2 above)
Comment (bcdunbar) [reply app.py]: If you like. I was only sweeping through dash repos and removing RawGit.

**Pull request merged**